gluetool/log.py

```diff
@@ -707,6 +717,7 @@ class LoggingFormatter(logging.Formatter):
     #: Colorizers assigned to loglevels
     _level_color = {
         logging.INFO: lambda text: Colors.style(text, fg='green'),
+        IMPORTANT: lambda text: Colors.style(text, fg='blue'),
```

Comment:
How about using `cyan` instead of `blue`? A blue text seems to me as hard to read on darker backgrounds (good on lighter) while cyan should work good enough on both black and white backgrounds.